commissions/models.py

```diff
@@ -1,16 +1,3 @@
 from django.db import models
```

Comment:
Wanted to delete the whole app in the same PR but due to some migration dependency headaches in users it was put on hold